tests/installation/select_visible_patterns.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Select given PATTERNS
+#    You can pass
+#    PATTERNS=minimal,base or
+#
+#    For this you need to have needles that provide base-pattern,
+#    minimal-pattern...
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+use utils 'type_string_slow';
+
+sub run {
```

Comment:
we could call this test module `select_only_visible_patterns_from_top` so it is clear that we clean the list and navigate from top. Same for the method name: `process_only_visible_patterns_from_top`.